test/vcf/record_cache_test.cpp

```diff
@@ -142,8 +153,43 @@ namespace ebi
         SECTION("Symbolic Duplicates")
         {
             CHECK( count_duplicates_and_rethrow_error(cache, {107, "C", {"<INS>"}}) == 0 );
-            // This won't throw an error. Only warning would be generated, which isn't detected here.
+            // This won't throw an error.
```

Comment:
This comments don't add information. `count_duplicates_and_rethrow_error` should only throw in case of inner malfunction. if there are no duplicates it returns 0, and if there are duplicates it returns the count. maybe the function should be called `count_duplicates_or_rethrow_error`, or `count_duplicates` to hide implementation details.